test/e2e-test-application/src/app/project/project.component.html

```diff
@@ -291,6 +291,17 @@ <h3 class="fd-layout-panel__title">Navigate</h3>
               data="linkManager().navigate('users/groups/stakeholders')"
             ></app-code-snippet>
           </li>
+          <li class="fd-list__item">
+            <a
+              href="javascript:void(0)"
+              class="fd-link"
+              (click)="linkManager().navigate('/projects/pr1/settings?ft=test')"
```

Comment:
This worked very well, just that after navigation finished you don't really notice what happens unless you query `getActiveFeatureToggleList`. Was thinking to add one more line to print/show a text if `ft=test` on the settings page as well, just like you do for ft1 🤔